### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by raphaeldieu**
User can create a form

### CODE REVIEW

File: app/assets/javascripts/angular/controller/forms.js.erb [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-
-}]);
-
-app.controller('FormsUpdateCtrl', ['$scope', '$http', '$auth', '$location', '$routeParams', function($scope, $http, $auth, $location, $routeParams){
+app.controller('ListFormsCtrl', ['$scope', '$http','$auth','$location','user', function($scope, $http, $auth, $location, user){
   
   url = "<%= ENV['URL'] %>" || 'http://localhost:3000'
 
-  $scope.form = {};
+  $scope.noForms = false;
```
Comment (h8rry) [inline app/assets/javascripts/angular/controller/forms.js.erb:5]: what does it mean by "noForms?"

File: app/assets/javascripts/angular/controller/forms.js.erb [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
 
-}]);
\ No newline at end of file
+  $scope.form = {};
+  $scope.createForm = function() {
+    $http.post(url + '/forms',$scope.form).success(function(response) {
+      console.log(response)
+      $location.path('forms/' + response.id + '/edit');
+    }).error(function(response){
+      alert("write something you douche");
```
Comment (h8rry) [inline app/assets/javascripts/angular/controller/forms.js.erb:20]: #douche

File: app/assets/javascripts/angular/controller/navbar.js.erb [hunk 3]
```diff
@@ -15,4 +15,14 @@ app.controller('NavbarCtrl', ['$scope', '$auth', '$location','$window', function
       console.log(response);
     });
   };
-}]);
\ No newline at end of file
+}]).
+factory('notify', ['$window', function(win) {
```
Comment (h8rry) [inline app/assets/javascripts/angular/controller/navbar.js.erb:19]: Where are you using this?

File: app/assets/javascripts/angular/controller/forms.js.erb (hunk 1 above)
Comment (h8rry) [inline app/assets/javascripts/angular/controller/forms.js.erb:5]: Maybe `hasForms` is a better name?

File: app/controllers/contents_controller.rb [hunk 4]
```diff
@@ -4,6 +4,7 @@ def index
   end
 
   def create
+    binding.pry
```
Comment (h8rry) [inline app/controllers/contents_controller.rb:7]: You can take this out

File: app/controllers/forms_controller.rb [hunk 5]
```diff
@@ -17,7 +17,7 @@ def update
       render json: {message: "Cannot find form"}, status: :not_found
     else
       if @form.update(form_params)
-        render json: {updated: true}, status: :success
+        render json: {updated: true}
```
Comment (h8rry) [inline app/controllers/forms_controller.rb:20]: Should give a status

**Comment by h8rry** [comment]
@raphaeldieu Please review

**Comment by ctlong** [comment]
Hey harry, so we realized that our other pr has all this stuff plus more, so we're closing this one. Check out the other one.

Ps. we removed the status stuff so that it wouldn't return an error every time.

**Pull request closed without merging**